config/lpdc-management/characteristics/form.ttl

```diff
@@ -1,4 +1,5 @@
 @prefix form: <http://lblod.data.gift/vocabularies/forms/> .
+@prefix fieldGroups: <http://data.lblod.info/field-groups/> .
```

Comment:
Not needed I think. i copied it from a v1 form but it doesn't seem to be used here.